compiler/src/dotty/tools/io/File.scala

```diff
@@ -0,0 +1,114 @@
+/*                     __                                               *\
```

Comment:
The commit message should include the hash of the version of theses files you're importing. If it works, try importing the latest version from scala 2.12.